apollo-compiler/src/test/graphql/com/example/fragment_with_inline_fragment/HeroDetailsExpected.java

```diff
@@ -0,0 +1,50 @@
+package com.example.fragment_with_inline_fragment;
+
+import java.lang.Integer;
+import java.lang.String;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public interface HeroDetails {
```

Comment:
it would be nice if we could add the original query string to the interface definition for reference (also we'll need that later). That will make it easier to review this. As these interfaces get more complicated, it's harder to spot bugs